[IMP] subscriptions: products fix misleading info (#5560)
odoo/documentation: content/applications/sales/subscriptions/products.rst

customer pays for (and receives) subscription products.

To configure recurrence periods, go to :menuselection:`Subscriptions app --> Configuration -->
Recurrence Periods`.

.. image:: products/recurrence-periods-page.png
   :align: center
   :alt: The recurrence periods page in Odoo Subscriptions application.

The *Subscriptions* app comes with some basic recurrence periods already configured:

- :guilabel:`Monthly`
- :guilabel:`Quartlery`
- :guilabel:`Weekly`
- :guilabel:`2 Weeks`
- :guilabel:`Yearly`
- :guilabel:`3 Years`
- :guilabel:`5 Years`

New recurrence periods can be added and/or edited at any time.

To create a new recurrence period, click :guilabel:`New` on the :guilabel:`Reccurence Periods` page.
Doing so reveals a blank recurrence period form.

.. image:: products/recurrence-period-form.png
   :align: center
   :alt: A reccurence period form in Odoo Subscriptions application.

Then, type in the :guilabel:`Name` and :guilabel:`Duration` of the recurrence period, and select the
:guilabel:`Unit` that defines the duration.

.. important::
   The unit :guilabel:`Days` *cannot* be used as a recurrence period on subscriptions. The daily

>    recurrence is meant for rentals, and *cannot* be added on recurring sales orders.

Reviewer: ```suggestion
   recurrence is meant for rentals, and **cannot** be added on recurring subscription sales orders.
```
I like that you added the call out to rentals. Since you are able to select the days unit from the drop-down, I just wanted to be as specific as possible as to why they shouldn't do it, but also "hey, there is an actual reason why that option is here, even if you shouldn't use it in this case" 😄

   This limitation is there to avoid sales orders that would generate daily invoices.

Product form configuration
==========================

> To create a new subscription product, start by navigating to the :menuselection:`Subscriptions` app.

Reviewer: ```suggestion
To create a new subscription product, navigate to :menuselection:`Subscriptions app --> Products --> Products`, and click :guilabel:`New`.
```

> Then, go to :menuselection:`Products --> Products`, and click :guilabel:`New` to create a new
> product. Doing so reveals a blank product form, which can be configured and customized in a number

Reviewer: ```suggestion
Doing so reveals a blank product form, which can be configured and customized in a number
```

of ways.

.. note::

>    By default, the :guilabel:`Recurring` option is already enabled, which enables Odoo to recognize

Reviewer: ```suggestion
   By default, the :guilabel:`Recurring` option is already enabled, prompting Odoo to recognize
```

   it as a subscription product. Be sure to leave the :guilabel:`Recurring` and :guilabel:`Can be
   Sold` options enabled.

   The :guilabel:`Product Type` field is set to :guilabel:`Service` by default, as well. However,
   subscription products *can* be set to other types, if needed.

.. image:: products/subscription-product-form.png
   :align: center
   :alt: A basic subscription product form in Odoo Subscriptions application.

Time-based pricing
------------------

Once the desired fields in the :guilabel:`General Information` tab have been entered, click the
:guilabel:`Time-based pricing` tab on the product form.

.. image:: products/time-based-pricing-tab.png
   :align: center
   :alt: The time-based pricing tab on a subscription product form in Odoo Subscriptions.

From here, click :guilabel:`Add a price` to begin defining recurring prices.

In the :guilabel:`Period` column, select a desired recurrence period. In the :guilabel:`Pricelist`
column, select a pricelist, if needed. Then, in the :guilabel:`Price` column, enter the price for
that recurrence period.

.. note::

>    :guilabel:`Daily` and :guilabel:`Hourly` periods *cannot* be used on recurring products.

Reviewer: ```suggestion
   :guilabel:`Daily` and :guilabel:`Hourly` periods **cannot** be used on recurring products.
```

When pricelists are added to the :guilabel:`Time-based pricing` tab, the pricelist form in the
*Sales* app is automatically updated.

Time-based pricing rules can also be configured directly on the pricelist form.

To do this, go to :menuselection:`Sales app --> Products --> Pricelists`, and select a pricelist (or
click :guilabel:`New` to create a new pricelist).

.. note::

>    This is also accessible through the Odoo *Subscriptions* app by following the same menu steps.

Reviewer 2: ```suggestion
   Pricelists are also accessible through the Odoo *Subscriptions* app by following the same menu steps.
```

When pricelists are added to the :guilabel:`Time-based pricing` tab, the pricelist form in the
*Sales* app is automatically updated.

Time-based pricing rules can also be configured directly on the pricelist form.

To do this, go to :menuselection:`Sales app --> Products --> Pricelists`, and select a pricelist (or

> click :guilabel:`New` to create a new pricelist).

Reviewer: May want to call out that this is also accessible through the Subscription app (follows the same menu steps)
